cdap-validation/bin/testing.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+
+import os
+import test_helpers as helpers
+import json
+import re
+from subprocess import call
+import ambari
+import cm
+
+##### TEST FRAMEWORK FUNCTIONS #####
+
+input_json = 'input.json'
```

Comment:
The idea here is that this gets replaced with command-line parsing/options and this file goes away.